doc_source/service-accounts.md

```diff
@@ -45,4 +45,5 @@ The following cluster add\-ons have been updated to use the Kubernetes client SD
 + CoreDNS version `1.8.4` and later\. To check your current version or update it, see [Managing the CoreDNS add\-on](managing-coredns.md)\.
 + AWS Load Balancer Controller version `2.0.0` and later\. To check your current version or update it, see [Installing the AWS Load Balancer Controller add\-on](aws-load-balancer-controller.md)\.
 + `kube-proxy` version `1.21.2-eksbuild.2` and later\. To check your current version or update it, see [Managing the `kube-proxy` add\-on](managing-kube-proxy.md)\.
-+ AWS for Fluent Bit version `2.25.0` or later\. To update your current version, see [Releases](https://github.com/aws/aws-for-fluent-bit/releases) on GitHub\.
\ No newline at end of file
++ AWS for Fluent Bit version `2.25.0` or later\. To update your current version, see [Releases](https://github.com/aws/aws-for-fluent-bit/releases) on GitHub\.
++ Fluentd plugin image version [1.14.6-1.2] (https://hub.docker.com/r/fluent/fluentd/tags?page=1&name=v1.14.6-1.2) or later, and Fluentd plugin version [2.11.1] (https://rubygems.org/gems/fluent-plugin-kubernetes_metadata_filter/versions/2.11.1) or later. 
```

Comment:
Fluentd plugins are a generic concept. You mean something like "Fluentd filter plugin for kubernetes metadata version 2.11.1"